modules/json/src/main/scala/gem/json.scala

```diff
@@ -25,6 +25,10 @@ package object json {
         b => Angle.fromMicroarcseconds(b.underlying.movePointLeft(6).longValue))(
         a => BigDecimal(new java.math.BigDecimal(a.toMicroarcseconds).movePointRight(6)))
 
+  // Wavelength mapping to integral Angstroms.
+  implicit val WavelengthCodec: CodecJson[Wavelength] =
+    CodecJson.derived[Int].xmap(Wavelength.unsafeFromAngstroms)(_.toAngstroms)
+
```

Comment:
Same comment as above, assuming we want JSON to be human-readable, which I think is a good starting position.